bot/src/discord/create_kanji_search_navigation_chapter.js

```diff
@@ -42,44 +57,24 @@ class KanjiNavigationDataSource {
   }
 }
 
-async function createKanjiSearchNavigationChapter(
+function createKanjiSearchNavigationChapter(
   searchQuery,
   authorName,
   commandPrefix,
   forceNavigationFooter,
 ) {
-  let kanji = extractKanji(searchQuery);
-  if (kanji.length === 0) {
-    kanji = await jishoKanjiSearch(searchQuery);
-  }
-
-  if (kanji.length === 0) {
-    const pages = [{
-      embed: {
-        title: 'Jisho Kanji Search',
-        description: `I didn't find any results for ${searchQuery}`,
-        color: constants.EMBED_NEUTRAL_COLOR,
-      },
-    }];
-
-    return {
-      navigationChapter: NavigationChapter.fromContent(pages),
-      pageCount: 1,
-      hasResult: false,
-    };
-  }
 
   const dataSource = new KanjiNavigationDataSource(
-    kanji,
+    searchQuery,
     authorName,
     commandPrefix,
     forceNavigationFooter,
   );
 
   return {
     navigationChapter: new NavigationChapter(dataSource),
-    pageCount: kanji.length,
```

Comment:
I have done as you suggested. But now I'm not quite sure how to get the `pageCount` and `hasResult` that is returned by `createKanjiSearchNavigationChapter`, without accessing the kanji page.  It seems that the real count of the page doesn't really matters. What seems to matter is only whether it is one or more than one page.